README.md

```diff
@@ -19,7 +19,7 @@ steps:
 ## Advanced Usage
 
 Install a specific version of the `oras` CLI by specifying the input `version` without the prefix `v`.
-Supported versions can be found at [`oras` releases](https://github.com/oras-project/oras/releases).
+Stable versions (not `beta` or `rc` versions) that can be found at [`oras` releases](https://github.com/oras-project/oras/releases) are supported.
```

Comment:
Suggest not to mentioned `beta` or `rc`, since there can also be `alpha`.
```suggestion
Stable versions (not pre-release versions) are supported and can be found at [`oras` releases](https://github.com/oras-project/oras/releases) are supported.
```